EXILED/Exiled.API/Features/Items/Armor.cs

```diff
@@ -74,12 +74,12 @@ internal Armor(ItemType type)
         }
 
         /// <summary>
-        /// Gets or sets a value indicating whether excess ammo should be removed when the armor is dropped.
+        /// Gets a value indicating whether excess ammo should be removed when the armor is dropped.
         /// </summary>
         public bool RemoveExcessOnDrop
         {
             get => !Base.DontRemoveExcessOnDrop;
-            set => Base.DontRemoveExcessOnDrop = !value;
+            internal set => Base.DontRemoveExcessOnDrop = !value;
```

Comment:
Why are you making this internal ?

Reply:
> Why are you making this internal ?

because that shouldnt be accessible, the game only uses this to avoid removing unnecessary items when clearing inventory, and changing it to any value is dangerous

Reply:
And exiled use that only in `ClearItems` method

Reply:
ah okay, i understood that can be public, nvm